app/src/main/java/me/hufman/androidautoidrive/carapp/music/TextScroller.kt

```diff
@@ -0,0 +1,52 @@
+package me.hufman.androidautoidrive.carapp.music
+
+/**
+ * Wrapper around a string that handles scrolling when it is too long to fit on a line.
+ */
+class TextScroller(private val originalText: String, private val maxLineLength: Int) {
+	companion object {
+		const val SCROLL_COOLDOWN_SECONDS = 6
+		const val INDEX_JUMP_VALUE = 3
+	}
+
+	val shouldScroll: Boolean
```

Comment:
The IDE suggests that these be private, which normally doesn't matter, but there could be a crash if startIndex gets modified to be longer than the string

Reply:
Good point, updated all these member variables to be private